content/_index.html

```diff
@@ -0,0 +1,66 @@
++++
+title = "FSH School"
+linkTitle = "FSH School"
+
++++
+
+{{< blocks/cover title="Welcome to FSH School" image_anchor="top" height="full" color="orange" >}}
+<div class="mx-auto">
+	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
+		Learn More <i class="fas fa-fish ml-2"></i>
+	</a>
+	<a class="btn btn-lg btn-secondary mr-3 mb-4" target="_blank" href="http://standardhealthrecord.org/fsh-playground/">
+		Experiment <i class="fas fa-flask "></i>
+	</a>
+	<p class="lead mt-5">The place to learn and try FHIR Shorthand</p>
+	{{< blocks/link-down color="white" >}}
+</div>
+{{< /blocks/cover >}}
+
+
+{{% blocks/lead color="primary" %}}
+FHIR Shorthand (FSH) is a domain-specific language for defining the contents of FHIR Resources and Implementation Guides.
+Explore FSH School to get the knowledge you need to get started using FSH.
+{{% /blocks/lead %}}
+
+
+{{< blocks/section color="white">}}
+{{% blocks/feature icon="far fa-file-alt" title="Declarative" %}}
+FSH allows you to define FHIR profiles in a simple, declarative language.
+No more large JSON files or spreadsheets, just human-readable text files
+which compile to the FHIR artifacts you need.
+{{% /blocks/feature %}}
+
+
+{{% blocks/feature icon="fab fa-github" title="Source control ready" %}}
+Unlike other tools for profiling FHIR, FSH works seamlessly with source control tools like
+GitHub. Maintain a history of changes, track contributions, and maintain agility by using FSH with source control.
+{{% /blocks/feature %}}
+
+
+{{% blocks/feature icon="fas fa-fire" title="State of the art" %}}
+The FSH Specification evoles as FHIR does, ensuring you have access to the latest and greatest features of FHIR. 
```

Comment:
Might some people think that this means FSH should work w/ not-yet-released FHIR versions?  For example, FHIR is currently evolving toward R5 (and has been for about a year now).  If FSH were really evolving w/ FHIR, wouldn't we support the currently evolving R5?  Or am I reading too much into this?

Reply:
I don't think you're reading too much into this, but a lot of this text on the front page was made under the assumption that you and @markkramerus may want to make edits to the content itself, so don't feel like having a "State of the art" section is necessarily required. You're free to ask for different sections altogether.